core/runtime/core_builtin.odin

```diff
@@ -591,7 +591,7 @@ assign_at_elems :: proc(array: ^$T/[dynamic]$E, index: int, args: ..E, loc := #c
 		copy(array[index:], args)
 		ok = true
 	} else {
-		resize(array, index+1+len(args), loc) or_return
+		resize(array, index+len(args), loc) or_return
```

Comment:
Does this take into the account when `len(args) == 0`?

Reply:
it works

Reply:
Thanks for quickly reviewing. it didn't but now it does